### DISCUSSION TITLE
Command not found

### DISCUSSION THREAD

**Discussion by HyperCodec** [Rapptz/discord.py#6062] [Q&A]
So, there's this thing where even though my script is there, it says "command dm not found"(I'm making an advertising bot for my premium users to use) and it only does it with the DM command, not any of the other ones.
```python
# import stuff
import discord
from discord.ext import commands
from discord.utils import get

# classify variables
client = commands.Bot(command_prefix = '+')
token = 'not telling u'

# say "bot is ready" when on
@client.event
async def on_ready():
     print('Bot is ready!')

# commands
@client.command()
async def ping(ctx):
    await ctx.send(f'The current bot speed is {round(client.latency * 1000)} ms')
@client.command()
async def dmall(ctx, *, message): # this is the command that doesn't work
    for user in list(ctx.guild.members):
        try:
            await asyncio.sleep(0.5)    
            await user.send(message)
            await ctx.send(f'Sent "{message}" To {user}')
        except:
            pass


# run bot :)
client.run(token)
```
![image](https://user-images.githubusercontent.com/72839119/99587308-ec4a2180-29b6-11eb-9e43-3fbeccbdd4d0.png)

**Comment by NESTOWL** [comment]
![image](https://user-images.githubusercontent.com/45520187/99597500-2217f600-2a09-11eb-911b-5ac8f63a3a75.png)

**Comment by mmcheng55** [comment]
You don't have a command called "dm". Instead you have a command called dmall. You should try calling dmall but not dm.
